custom/ilsgateway/tests/handlers/arrived.py

```diff
@@ -1,38 +0,0 @@
-from custom.ilsgateway.tanzania.reminders import ARRIVED_HELP, ARRIVED_DEFAULT, ARRIVED_KNOWN
-from custom.ilsgateway.tests.handlers.utils import ILSTestScript
-
-
-class ILSArrivedTest(ILSTestScript):
```

Comment:
Yeah that's correct. The benefit here is from running the setup once instead of 20x. Although if https://github.com/dimagi/commcare-hq/pull/14840 fixes prepare_domain then it's possible that this won't have as large of an effect as noted above